external/process-cpp-minimal/src/core/posix/signal.cpp

```diff
@@ -203,12 +203,10 @@ class SignalTrap : public core::posix::SignalTrap
 };
 }
 
-std::shared_ptr<core::posix::SignalTrap> core::posix::trap_signals_for_process(
-        std::initializer_list<core::posix::Signal> blocked_signals)
+core::posix::SignalTrap* core::posix::trap_signals_for_process(
```

Comment:
No need to change the formatting here

Reply:
This format is changed because the cycling-reference of shared_ptr is checked in valgrind, I replace it with pointer with explicit new and delete. This time the valgrind doesn't complain it.

Reply:
I've reverted this commit.